### PR TITLE
Network notifications

### PR DESCRIPTION
**Pull request by namedotget**
- Deployed new job and maretkplace contracts that emit events
- Added discord network notifications to /TeamJobModal, /TeamMaretkpaceListingModal, /CreateCitizen, /CreateTeam
- Scroll to job when jobId is in team url
- Adjusted metadata of team page, show job or listing data if in url

### CODE REVIEW

File: ui/components/onboarding/CreateTeam.tsx [hunk 1]
```diff
@@ -459,10 +465,22 @@ export default function CreateTeam({
...
+                          const teamNFT = await waitForERC721(
+                            teamContract,
+                            mintedTokenId
+                          )
+                          const teamName = teamNFT?.metadata.name as string
+                          const teamPrettyLink = generatePrettyLink(teamName)
+                          await sendDiscordMessage(
+                            accessToken,
+                            'networkNotifications',
+                            `[**${teamName}** has minted a team NFT!](${DEPLOYED_ORIGIN}/team/${teamPrettyLink}?_timestamp=123456789)`
```
Comment (jaderiverstokes) [inline ui/components/onboarding/CreateTeam.tsx:478]: ? timestamp?

File: ui/components/subscription/TeamJobModal.tsx [hunk 2]
```diff
@@ -132,6 +150,24 @@ export default function TeamJobModal({
               ])
             }
 
+            //Get job id and team id from receipt and send discord notification
+            const jobId = parseInt(tx.receipt.logs[1].topics[1], 16).toString()
```
Comment (jaderiverstokes) [inline ui/components/subscription/TeamJobModal.tsx:154]: FYI: i think something like this should work

const iface = new ethers.utils.Interface(JobsABI);
const parsedLog = iface.parseLog(tx.receipt.logs[1]);
const jobId = parsedLog.args.jobId
const jobTeamId = parsedLog.args.job

**Review by jaderiverstokes — APPROVED** [review]
did you ever commit the new contracts with events? i didn't see them maybe i just missed it.

File: ui/components/onboarding/CreateCitizen.tsx [hunk 3]
```diff
@@ -503,12 +507,25 @@ export default function CreateCitizen({
...
+                        )
+                        const citizenName = citizenNFT?.metadata.name as string
+                        const citizenPrettyLink = generatePrettyLinkWithId(
+                          citizenName,
+                          mintedTokenId
+                        )
+                        await sendDiscordMessage(
+                          accessToken,
+                          'networkNotifications',
+                          `[**${citizenName}** has minted a citizen NFT!](${DEPLOYED_ORIGIN}/citizen/${citizenPrettyLink}?_timestamp=123456789)`
```
[the remark this answers is not in the document]
Comment (namedotget) [reply ui/components/onboarding/CreateCitizen.tsx:523]: Yes, it allows the metadata of the link to refresh

**Comment by namedotget** [comment]
> did you ever commit the new contracts with events? i didn't see them maybe i just missed it.

Will push

**Pull request merged**